.github/workflows/build-verification.yml

```
@@ -45,25 +45,25 @@ jobs:
           - 6334:6334
         # Remove health check for now to let the container start properly
         # We'll check connectivity in a separate step
-    
+
     steps:
     - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)